File: Java/question2.java
import java.util.*;
class question2{
  public static void main(String args[]){
    Scanner input = new Scanner(System.in); // 輸入工具
    int times = input.nextInt(); // 輸入次數
    int total = 0;
    for(int i = 0 ; i < times ; i++){
      total = total + input.nextInt();
    }
    float answer = total/times;
    System.out.printf("%.3f",answer);
  }
}

File: Python/question2.py
"""
 2. 測試範例 平均
"""

times = int(input()) ## 輸入次數
total = 0
for i in range(times):
  total = total + int(input())

print("{:.3f}".format(total/times)) ## .3f是用來顯示小數點到第三位的